src/actions/sso/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { IssuerPublicProfileSummary } from './types'
+import { Identity } from 'jolocom-lib/js/identity/identity'
+
+export const parsePublicProfile = (identity: Identity) => {
```

Comment:
This could use documentation, i.e. a comment describing it's purpose and arguments.

Reply:
Added